README.md

```diff
@@ -43,6 +43,7 @@ The React render happens on both the server and the client using shared code. Re
 
 ```
 npm install
+npm run build
```

Comment:
I don't think build is necessary when you start server. Build is done in runtime.